### ISSUE TITLE
flask error

### ISSUE THREAD

**Issue by ims1337** [bastien8060/MDPin#2]
I see below error while i run:
```
~/tools/MDPin/py# python3 pwn.py 
Traceback (most recent call last):
  File "pwn.py", line 2, in <module>
    from flask_restful import Resource, Api
ModuleNotFoundError: No module named 'flask_restful'
```

**Comment by bastien8060** [comment]
Oh it seems it isn't installed. Maybe try to install it via: `pip install flask-restful` or `pip3 install flask-restful`. It seems you are using GNU/Linux right? In this case, you might need sudo.

**Edit:** Sometimes it might need `Flask_RESTful` instead of `flask-restful` because of the special case.

**Comment by bastien8060** [comment]
By the way, have you installed Flask? If not, you might want to run: `pip install Flask`. 

If you use Linux, pip might need sudo, but not necessarly.

**Comment by bastien8060** [comment]
If you redownload the latest update I just made, `flask_restful` and other dependencies you are missing will download and install automatically

**Comment by ims1337** [comment]
Thanks for quick reply.

I am using Shell based UBUNTU.

No i dont have flask. I'll be retrying this one again.

Let me try again

**Comment by bastien8060** [comment]
Okay sure :)

Tell me if it works so I can set this issue as resolved or fix something further along.

**Comment by bastien8060** [comment]
Is it fixed? Tell me whenever you want

**Comment by bastien8060** [comment]
I'm closing this issue now due to inactivity. No reply was given since 48h, so I'm assuming this issue was fixed (Hoping!)  :)
